src/server/auth/cmds/cmds.go

```diff
@@ -380,7 +380,7 @@ func WhoamiCmd() *cobra.Command {
 			}
 			fmt.Printf("You are \"%s\"\n", resp.Username)
 			if resp.Expiration != nil {
-				fmt.Printf("session expires: %v\n", *resp.Expiration)
+				fmt.Printf("session expires: %v\n", resp.Expiration.Format(time.RFC822))
```

Comment:
It's not worth pulling this out, but for unrelated small changes in the future it's better to make a separate PR. It makes understanding the history easier, and reverting things cleaner.